src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsaTransparency.java

```diff
@@ -16,7 +16,7 @@ public class ExtRegsDsaTransparency {
     String domain;
 
     /**
-     * Defines the contract for bidrequest.regs.ext.dsa.transparency[i].params[]
+     * Defines the contract for bidrequest.regs.ext.dsa.transparency[i].dsaparams[]
      */
-    List<Integer> params;
+    List<Integer> dsaparams;
```

Comment:
Please follow Java code style and use @JsonProperty or use Jackson naming strategy